src/core/ddsi/src/q_xmsg.c

```diff
@@ -1217,7 +1217,7 @@ void nn_xpack_sendq_init (struct q_globals *gv)
 
 void nn_xpack_sendq_start (struct q_globals *gv)
 {
-  create_thread (&gv->sendq_ts, gv, "sendq", nn_xpack_sendq_thread, NULL);
+  (void) create_thread (&gv->sendq_ts, gv, "sendq", nn_xpack_sendq_thread, NULL);
```

Comment:
This is some code that is more of an experiment than something that should remain. Still, the response to Coverity complaining that the result of attempting to create a thread is being ignored should not be to tell it to shut up in this way.

And should it be determined that this is, all considered, the right approach, it warrants a little note why it is so.

Reply:
Will check the return code and log a message.